bodhi/server/views/generic.py

```diff
@@ -208,7 +208,7 @@ def work():
         critpath_updates = get_latest_updates(request, True, False)
```

Comment:
This function is cached - that might be the cause of the test isolation problems.